code/modules/wiremod/components/string/tostring.dm

```diff
@@ -34,11 +32,7 @@
 		return
 
 	var/input_value = input_port.input_value
-	if(isatom(input_value))
-		var/turf/location = get_turf(src)
-		var/atom/object = input_value
-		if(object.z != location.z || get_dist(location, object) > max_range)
-			output.set_output(PORT_TYPE_ATOM)
-			return
 
 	output.set_output("[input_value]")
+
+//MONKESTATION EDIT: Removed max range, makes sense on some other components, but not this one
```

Comment:
Put the max range back in, but with a more reasonable limitation.
No user input should be unlimited.